django/conf/global_settings.py

```diff
@@ -263,6 +263,12 @@ def gettext_noop(s):
 # loudly.
 SECRET_KEY = ''
 
+# A list of secret keys that can be used in addition to the SECRET_KEY to
+# verify signatures.
+OLD_SECRET_KEYS = []
+
+SECRET_KEY_BACKEND = 'django.core.secret_key.SecretKeySettingsBackend'
```

Comment:
`SECRET_KEYS_BACKEND` probably better

Reply:
Not relevant anymore since this PR no longer contains the settings backend stuff. :)